docs/basics/changelog/1_0_51.md

```diff
@@ -0,0 +1,87 @@
+# API Changelog for Apr 2022 (v.1.0.51)
+
+### Voice
+
+### Analytics
+
```

Comment:
Do we need to keep these, or can they be removed?

Reply:
removed 'Analytics' as it's also in subheading but kept 'Voice' as it's top-level category